torch/nn/functional.py

```diff
@@ -406,6 +407,28 @@ def dropout(input, p=0.5, training=False, inplace=False):
     return _functions.dropout.Dropout(p, training, inplace)(input)
 
 
+def alpha_dropout(input, p=0.5, training=False):
+    if p < 0 or p > 1:
+        raise ValueError("dropout probability has to be between 0 and 1, "
+                         "but got {}".format(p))
+
+    if p == 0 or not training:
+        return input
+
+    alpha = -1.7580993408473766
```

Comment:
this looks like alpha prime = - lambda * alpha from the paper (arxiv:1706.02515). Why is this a fixed value (in the paper it looks like it depends on the preceding SELU activation layer) and what expression leads to this numerical value ?